src/Windows/Perspex.Win32/Interop/UnmanagedMethods.cs

```diff
@@ -751,6 +754,8 @@ internal class FileOpenDialogRCW { }
         [DllImport("comdlg32.dll")]
         public static extern int CommDlgExtendedError();
 
+        public static bool ShCoreAvailable => LoadLibrary("shcore.dll") != IntPtr.Zero;
```

Comment:
I'm wondering if this value should be cached instead of calling `LoadLibrary` each time?

The documentation for `LoadLibrary` says:

> The system maintains a per-process reference count on all loaded modules. Calling LoadLibrary increments the reference count. Calling the FreeLibrary or FreeLibraryAndExitThread function decrements the reference count. The system unloads a module when its reference count reaches zero or when the process terminates (regardless of the reference count).

To me, this suggests that we shouldn't call it repeatedly.